dist/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <title>The View | JS Capstone</title>
+<script defer src="runtime.bundle.js"></script><script defer src="main.bundle.js"></script></head>
+
+<body>
+  <div id="app">
+    <header>
+      <img class="logo" src="assets/images/the-view-logo.png" alt="logo">
+      <h1 class="page-title">Popular Movies (6)</h1>
```

Comment:
You should implement a counter for the number of items on the home screen.

<img width="855" alt="Screen Shot 2022-07-29 at 8 28 59" src="https://user-images.githubusercontent.com/82426682/181770635-95894211-0a8d-47d4-93d1-ac29bbf344d7.png">

After you implement it, don't forget you need to test it as well.

Reply:
Counter is implemented in the js file. It is dynamic and not static